code-challenges.js

```diff
@@ -40,24 +79,72 @@ var vowelTester1 = "learn"
 // Expected output: 1
 var vowelTester2 = "throw"
 // Expected output: 3
-
-
-
-
-
+//Create a function that iterates each letters
+//have another function intandem to return the index of only the first vowel indexOf()
+//turn string into array of letters
+//for loop that checks AEIOU
+//return index
+var arrayVowel = ["e" , "a" , "o" , "u" , "i"]
+
+const indexr = (string) => {
+  let stringSplit = string.split("")
+  //console.log(stringSplit);
+  for(let i=0; i<stringSplit.length; i++){
+    if (arrayVowel.includes(stringSplit[i]))
+    return i
+  }
+  //return stringSplit
+  //return stringSplit.indexOf(value === "e" && "a" && "i" && "o" && "u")
+  //return stringSplit.indexOf("e" || "a" || "i" || "o" || "u")
+}
+console.log(indexr(vowelTester1));
+console.log(indexr(vowelTester2));
+
+/*
+  let vowelFilter = string.map(value )
+}
+
+const indexMeVowel = (string) => {
+  let vowelFilter = string.indexOf(value => {
+    return value === "a" && "e" && "i" && "o" && "u"})
+}
+console.log();
+console.log(indexMeVowel(vowelTester1));
+*/
 
 // --------------------5) Create a function that takes three arguments - two numbers and a mathematical operation (+, -, *, /) and performs the given calculation. If the input tries to divide by 0, return: "Can't divide by 0!"
-
-
-
-
-
+//                                               2       /         2
+// build a function that takes three arguments (num1, (+,-,*,/), num2 )
+// if/ else: if num1 & num2 is NOT !== divided by 0 , the return "Can't divide by 0!"
+/*}else{ if num1 & num2 is equal === divide by 0, return the operation.
+} */
+//declare our variables
+const calculator = (num1, symbol, num2) => {
+  //if else
+  //make variables for all symbols and have dividing by zero catch function first.
+  if(symbol === "+"){
+    return (num1 + num2)
+  }else if(symbol === "-"){
+    return (num1 - num2)
+  }else if(symbol === "*"){
+    return (num1 * num2)
+  }else if(symbol === "/" && num1 || num2 === 0){
+    return "Cant divide by 0!"
+  }else{(symbol === "/")
```

Comment:
This, else, does not work. Else does not take a condition, I think you are looking for an Else if.